src/components/ErrorMessage.tsx

```diff
@@ -84,17 +85,6 @@ ErrorMessage.displayName = 'ErrorMessage'
 ErrorMessage.defaultProps = {
     timeout: 2500,
 }
-ErrorMessage.propTypes = {
-    /** Message to be displayed. This component only renders when message is set. */
-    message: PropTypes.oneOfType([
-        PropTypes.string,
-        PropTypes.arrayOf(PropTypes.node),
-        PropTypes.node,
-    ]),
-    /** Timeout after the error message disappears (in ms). */
-    timeout: PropTypes.number,
-    /** Optional callback that is invoked when the error message disappears. */
-    onHide: PropTypes.func,
-}
 
+export { Props, State }
```

Comment:
As I already wrote somewhere, I think `Props` types should not be exported. They can always be extracted with `React.ComponentProps` helper.

State is internal so it also should not be exported?

Reply:
Done!